packages/integration/package.json

```diff
@@ -65,39 +65,39 @@
 	},
 	"homepage": "https://github.com/cydran/cydran",
 	"devDependencies": {
-		"@babel/cli": "7.23.4",
-		"@babel/core": "7.23.7",
-		"@babel/plugin-transform-modules-umd": "7.23.3",
-		"@babel/preset-env": "7.23.8",
-		"@babel/preset-typescript": "7.23.3",
-		"@testing-library/dom": "9.3.4",
-		"@testing-library/jest-dom": "6.3.0",
-		"@types/jest": "29.5.11",
-		"@types/jsdom": "21.1.6",
-		"@types/lodash": "4.14.202",
-		"@types/node": "20.11.6",
-		"@typescript-eslint/eslint-plugin": "7.3.1",
-		"@typescript-eslint/parser": "7.5.0",
+		"@babel/cli": "^7.25.6",
+		"@babel/core": "^7.25.2",
```

Comment:
These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

Reply:
fixing